testing/src/cypress/e2e/data/institution/manage-location.json

```diff
@@ -23,23 +23,34 @@
       "abc@",
```

Comment:
Are these `data` files serving two different purposes? Why do we have some on json and some on ts files?
What is the goal?

Reply:
The intention to have all the "valid and invalid" data combinations that needs to be tested against the input fields in on place - `manage-location.json`

Reply:
Sorry, but it still does not answer the question. It can be achieved with a JSON or a ts file and apparently both ways are used.